modules/claims_api/spec/controllers/v2/veterans/power_of_attorney/request_controller_spec.rb

```diff
@@ -55,6 +55,87 @@
         end
       end
     end
+
+    context 'when pageIndex is present but pageSize is not' do
+      before do
+        allow(subject).to receive(:form_attributes).and_return({ 'poaCodes' => %w[002 003 083], 'pageIndex' => '2' })
+      end
+
+      it 'raises a ParameterMissing error' do
+        expect do
+          subject.index
+        end.to raise_error(Common::Exceptions::ParameterMissing)
+      end
+    end
```

Comment:
missed this test in https://github.com/department-of-veterans-affairs/vets-api/pull/19268, so adding here.